internal/pkg/deploy/cloudformation/stack/static_site.go

```diff
@@ -101,6 +101,10 @@ func (s *StaticSite) Template() (string, error) {
 			return "", err
 		}
 	}
+	staticSiteAlias := fmt.Sprintf("%s.%s.%s.%s", s.name, s.env, s.app, s.appInfo.Domain)
```

Comment:
Just a note: we have NLB, apprunner, LBWS default alias implemented in a different way of using `AWS::Route53::RecordSetGroup` instead of setting it as an alias here by default. I am not leaning towards making it consistent because I don't believe that one is better.

However, maybe it's worth it to make a comment here so that people will know how static site set default alias is unique. And add an e2e test to capture this behavior.

Reply:
Because CF has a default alias already and we think users can just use that?

Reply:
right, but why don't we just do something like:
```
{{if .StaticSiteAlias}}
  AWS::Route53::RecordSetGroup
{{end}}
```
?

Reply:
Because we could've just use custom resource to manage the A-record so that we don't have two resources doing the same thing. Also `AWS::Route53::RecordSetGroup` won't delete the resource and swallow the error when you make any modification to its setting (for example, you updated its TTL). Leaving the A-record dangling is extremely annoying (because you can't delete the hosted zone then).